### ISSUE TITLE
Error Occurred When Deploy Schema: bopomofo

### ISSUE THREAD

**Issue by chenrylee** [rime/rime-bopomofo#7]
When Deploying schema: bopomofo, it said:
> Error occurred. See log file $TMPDIR/rim....

After removing this schema and related `yaml` files, the error still exists.

By checking logs:
```bash
E0307 14:57:18.708663 127107072 deployment_tasks.cc:319] Error updating schema: nonexistent file '/Users/username/Library/Rime/stroke.schema.yaml'.
E0307 15:09:56.349274 128180224 deployment_tasks.cc:319] Error updating schema: nonexistent file '/Users/username/Library/Rime/cangjie5.schema.yaml'.
```

By checking schema files:
```yaml
# bopomofo.schema.yaml
  dependencies:
    - cangjie5
reverse_lookup:
  dictionary: cangjie5
```

**Comment by chenrylee** [comment]
Update `README.md` via PR #6

**Comment by lotem** [comment]
Fixed in librime 1.5.0
